feedthefox/settings.py

```diff
@@ -98,6 +99,7 @@
 
 MEDIA_ROOT = config('MEDIA_ROOT', default=os.path.join(BASE_DIR, 'media'))
 MEDIA_URL = config('MEDIA_URL', '/media/')
+USER_PHOTOS_DIR = config('USER_PHOTOS_DIR', 'uploads/profiles')
```

Comment:
It isn't a good idea not to use ``/media`` for deployment/sanity reasons. I get your point about structure. Please add this folder under /media eg:

``MEDIA_ROOT/uploads/profiles``